packages/editor/src/components/block-toolbar/style.scss

```diff
@@ -1,5 +1,7 @@
 .editor-block-toolbar {
-	display: flex;
+	// Should be inline-flex so it isn't full-wide. 
+	// We leverage this to center the toolbar on wide and full-wide images.
+	display: inline-flex;
```

Comment:
This should be `flex` though to fix the toolbar in mobile in several browsers. I think we should leave it flex here and try to override the `editor-context-toolbar` instead for wide alignments.

Reply:
That could work too. So to understand you correctly, this was not done as part of IE11 fixes? 

I'll push fixes for wide and fullwide in a minute.

Reply:
It was but I think it affected several browsers, I'm testing right now to confirm